container.go

```diff
@@ -24,6 +24,7 @@ type containerConfig struct {
 	Broker              string `default:"amqp://localhost:5672"`
 	RootRepositoriesDir string `default:"/tmp/root-repositories"`
 	Locking             string `default:"local:"`
+	HDFSUrl             string `default:""`
```

Comment:
then, the env variable will be `CONFIG_HDFSURL` instead of `CONFIG_HDFS_URL`. Do we want this?